README.md

```diff
@@ -4,29 +4,34 @@
 
 You can use it as a Github Action like this:
 
-_.github/main.workflow_
+_.github/workflows/push.yml_
 ```
-workflow "Main" {
-  on = "push"
-  resolves = ["PHP-CS-Fixer"]
-}
-
-action "PHP-CS-Fixer" {
-  uses = "docker://oskarstark/php-cs-fixer-ga"
-}
+on: push
+name: Main
+jobs:
+  php-cs-fixer:
```

Comment:
```suggestion
  PHP-CS-Fixer:
```

Reply:
The github migration tools was going for `pHP-CS-Fixer:` so I figured that key should be lower case at least for the first character, hence the lowercase.
Do you think we can leave it as is (lower)?

Reply:
> Do you think we can leave it as is (lower)?

I would say yes 👍